static/index.html

```diff
@@ -4,7 +4,34 @@
         <meta charset="utf-8">
         <meta content="width=device-width, initial-scale=1" name="viewport">
         <title>ankerctl</title>
-        <link href="/static/vendor/bootstrap-5.3.0-alpha3-dist/css/bootstrap.min.css" rel="stylesheet">
+        <link href="{{ url_for('static', filename='vendor/bootstrap-5.3.0-alpha3-dist/css/bootstrap.min.css')}}" rel="stylesheet">
+        <link href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.4/font/bootstrap-icons.css" rel="stylesheet">
+        <script type="text/javascript" src="{{ url_for('static', filename='vendor/bootstrap-5.3.0-alpha3-dist/js/bootstrap.bundle.min.js')}}"></script>
+        <script type="text/javascript" src="{{ url_for('static', filename='vendor/cash.min.js')}}"></script>
+        <script type="text/javascript" src="{{ url_for('static', filename='vendor/jmuxer.min.js')}}"></script>
+        <script type="text/javascript" src="{{ url_for('static', filename='ankersrv.js')}}"></script>
+        <script type="text/javascript">
+         $(function () {
+             $('#configData').on('click',function(){
+                 navigator.clipboard.writeText("{{ configHost }}:{{ configPort }}");
+                 console.log('Copied {{ configHost }}:{{ configPort }} to clipboard...')
+                 return false;
+             });
+
+             $('#copyFilePath').on('click',function(){
+                 navigator.clipboard.writeText("{{ loginFilePath|replace('\\', '\\\\') }}");
+                 console.log('Copied {{ loginFilePath }} to clipboard...')
+                 return false;
+             });
+         });
+        </script>
+        <style>
+         .nav-link:hover {
```

Comment:
This seems a bit random. What is it for, and why is it inline?

If we need it, please create `static/ankersrv.css`, and put the declarations there.